hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/cachedigestion/SamplesInfoFileField.java

```diff
@@ -52,15 +55,15 @@ static SamplesInfoFileField getEnum(String fieldString)
 	}
 
 	/**
-	 * Compares the name of this {@link Enum} (excluding the {@link Enum} type name) to a {@link String}. Ignores
+	 * Compares the name of this {@link Enum} (excluding the {@link Enum} class name) to a {@link String}. Ignores
 	 * capitalization during the comparison!
 	 * 
 	 * @param fieldName
 	 *            {@link String}
-	 * @return {@code true} if the {@link String} equals the name of the {@link Enum} (ignoring the {@link Enum} type
+	 * @return {@code true} if the {@link String} equals the name of the {@link Enum} (ignoring the {@link Enum} class
 	 *         name), otherwise false.
 	 */
-	boolean nameEquals(String fieldName)
+	public boolean nameEquals(String fieldName)
 	{
 		return this.toString().equals(fieldName.toUpperCase());
```

Comment:
You could add a constructor for this enum class, which sets an instance variable 'name'. You can then list the enums together with their string names like so:

```java
EXTERNALSAMPLEID("externalSampleID"),
SEQUENCER("sequencer"), etc.
```

This removes the need to override ```getName()``` (which will just be a normal getter that returns the name), and make it a bit clearer which enum has which name.